doc/polyglossia.tex

```diff
@@ -44,14 +43,77 @@
 \def\Ie{\textit{I.e.,}\xspace}
 \def\etc{\@ifnextchar.{\textit{etc}}{\textit{etc.}\@\xspace}}
 
+%% Commands for documenting options
+\ExplSyntaxOn
+
+\NewDocumentCommand \pgoption {m}
+  {
+    \textcolor {pgblue} { \bfseries #1 }
+  }
+
+\NewDocumentCommand \pgdefault {m}
+  {
+    \textit{#1}
+  }
+
+\NewDocumentCommand \pgdefaultchoice {m}
+  {
+    \underline{#1}
+  }
+
+\NewDocumentCommand \pgchoicekey {omom}
+  {
+    \pgoption {#2}
+    \IfValueT {#3} { \footnote {#3} }
+    \IfValueT {#1} { \new {#1} }
+    \c_space_tl = \c_space_tl
+    \clist_set:Nn \l_tmpa_clist {#4}
+    \seq_set_from_clist:NN \l_tmpa_seq \l_tmpa_clist
+    \seq_use:Nnnn \l_tmpa_seq {~or~} {,~} {,~or~}
+    \par
+  }
+
+\NewDocumentCommand \pgboolkey {omo}
+  {
+    \pgchoicekey [#1] {#2} [#3] { \pgdefaultchoice{true}, false }
+  }
+ 
+\NewDocumentCommand \pgboolkeytrue {omo}
+  {
+    \pgchoicekey [#1] {#2} [#3] { \pgdefaultchoice{\pgdefault{true}}, false }
+  }
+
+\NewDocumentCommand \pgboolkeyfalse {omo}
+  {
+    \pgchoicekey [#1] {#2} [#3] { \pgdefaultchoice{true}, \pgdefault{false} }
+  }
+
+\NewDocumentCommand \pgbabelshorthands {o}
```

Comment:
I wouldn't introduce a separate macro for this. It's a simple bool key.

Reply:
Yes, but it occurs very often. This is a convenient shorthand, that also makes sure that the formatting is the same in all cases.

Reply:
I still think this is overload.

Reply:
I'm a bit surprised here. It is a basic principle of good programming to use a function for code that is needed repeatedly. But I can of course replace it with the more basis command.